app/models/energy_tariff_price.rb

```diff
@@ -18,14 +18,16 @@
 #
 class EnergyTariffPrice < ApplicationRecord
   MINIMUM_VALUE = 0.0
+  MAXIMUM_VALUE = 1.0
```

Comment:
Not something for this PR - but I wonder if this maximum value threshold could be a site setting?
Esp since this now stops the tariff from being saved (rather than warns) and likely needs to be reviewed periodically.

Reply:
Good point, thanks! I'll pop it in a Trello card.